### PR TITLE
Debugger/Disasm: Name SPU Syscalls

### PR DESCRIPTION
**Pull request by elad335**
Display SPU syscall names in the instructions' viewer.

### CODE REVIEW

File: rpcs3/Emu/Cell/lv2/sys_spu.cpp [hunk 1]
```diff
@@ -48,6 +48,25 @@ void fmt_class_string<spu_group_status>::format(std::string& out, u64 arg)
...
+{
+	format_enum(out, arg, [](spu_stop_syscall value)
+	{
+		switch (value)
+		{
+		case SYS_SPU_THREAD_STOP_YIELD: return "sys_spu_thread_yield";
+		case SYS_SPU_THREAD_STOP_GROUP_EXIT: return "sys_spu_thread_group_exit";
+		case SYS_SPU_THREAD_STOP_THREAD_EXIT: return "sys_spu_thread_thread_exit";
+		case SYS_SPU_THREAD_STOP_RECEIVE_EVENT: return "sys_spu_thread_receive_event";
+		case SYS_SPU_THREAD_STOP_TRY_RECEIVE_EVENT: return "sys_spu_thread_tryreceive_event";
```
Comment (Megamouse) [inline rpcs3/Emu/Cell/lv2/sys_spu.cpp:62]:
```suggestion
		case SYS_SPU_THREAD_STOP_TRY_RECEIVE_EVENT: return "sys_spu_thread_try_receive_event";
```

File: rpcs3/Emu/Cell/lv2/sys_spu.cpp [hunk 2]
```diff
@@ -48,6 +48,25 @@ void fmt_class_string<spu_group_status>::format(std::string& out, u64 arg)
...
 }
 
+template <>
+void fmt_class_string<spu_stop_syscall>::format(std::string& out, u64 arg)
+{
+	format_enum(out, arg, [](spu_stop_syscall value)
+	{
+		switch (value)
+		{
+		case SYS_SPU_THREAD_STOP_YIELD: return "sys_spu_thread_yield";
```
Comment (Megamouse) [inline rpcs3/Emu/Cell/lv2/sys_spu.cpp:58]: Shouldn't there be a stop_ in between?

File: rpcs3/Emu/Cell/lv2/sys_spu.cpp (hunk 1 above)
Comment (elad335) [reply rpcs3/Emu/Cell/lv2/sys_spu.cpp:62]: Intended.

File: rpcs3/Emu/Cell/lv2/sys_spu.cpp (hunk 2 above)
Comment (elad335) [reply rpcs3/Emu/Cell/lv2/sys_spu.cpp:58]: No.

File: rpcs3/Emu/Cell/lv2/sys_spu.cpp (hunk 1 above)
Comment (elad335) [reply rpcs3/Emu/Cell/lv2/sys_spu.cpp:62]: https://github.com/RPCS3/rpcs3/blob/7bb830a2285b2ce67489a16bfefc738e57251c19/rpcs3/Emu/Cell/SPUThread.cpp#L4495

**Pull request merged**